src/SonarQube.Client/Models/QualityProfiles.cs

```diff
@@ -0,0 +1,98 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using Newtonsoft.Json;
+
+namespace SonarQube.Client.Models
+{
+    public class QualityProfile
+    {
+        // Ordinal comparer, similar to project key comparer
+        public static readonly StringComparer KeyComparer = StringComparer.Ordinal;
+
+        public string Key { get; }
+        public string Name { get; }
+        public string Language { get; }
+        public bool IsDefault { get; }
+        public DateTime TimeStamp { get; }
+
+        public QualityProfile(string key, string name, string language, bool isDefault, DateTime timeStamp)
+        {
+            Key = key;
+            Name = name;
+            Language = language;
+            IsDefault = isDefault;
+            TimeStamp = timeStamp;
+        }
+
+        public static QualityProfile FromDto(QualityProfileDTO dto, DateTime timeStamp)
+        {
+            return new QualityProfile(dto.Key, dto.Name, dto.Language, dto.IsDefault, timeStamp);
+        }
+    }
+
+    public class QualityProfileDTO
+    {
+        [JsonProperty("key")]
+        public string Key { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("language")]
+        public string Language { get; set; }
+
+        [JsonProperty("isDefault")]
+        public bool IsDefault { get; set; }
+    }
+
+    public class QualityProfileChangeLogEventDTO
+    {
+        [JsonProperty("date")]
+        public DateTime Date { get; set; }
+    }
+
+    public class QualityProfileChangeLogDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'QualityProfileChangeLogDTO' to match camel case naming rules, consider using 'QualityProfileChangeLogDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)